src/items.ts

```diff
@@ -1395,133 +1440,168 @@ export const ITEMS = {
+    "Screen XL": otherElement("Screen XL", Category.INTERACTIVE, 11174.8),
+    "Transparent Screen XS": otherElement("Transparent Screen XS", Category.INTERACTIVE, 15.8),
+    "Transparent Screen S": otherElement("Transparent Screen S", Category.INTERACTIVE, 15.8),
+    "Transparent Screen M": otherElement("Transparent Screen M", Category.INTERACTIVE, 15.8),
+    "Transparent Screen L": otherElement("Transparent Screen L", Category.INTERACTIVE, 15.8),
+    "Sliding Door S": otherElement("Sliding Door S", Category.INTERACTIVE, 102.4),
+    "Sliding Door M": otherElement("Sliding Door M", Category.INTERACTIVE, 151.8),
+    "Encampment Chair": otherElement("Encampment Chair", Category.INTERACTIVE, 2),
+    "Navigator Chair": otherElement("Navigator Chair", Category.INTERACTIVE, 24),
+    "Office Chair": otherElement("Office Chair", Category.INTERACTIVE, 13),
+    "Long Light S": otherElement("Long Light S", Category.INTERACTIVE, 21.6),
+    "Long Light M": otherElement("Long Light M", Category.INTERACTIVE, 21.6),
+    "Long Light L": otherElement("Long Light L", Category.INTERACTIVE, 21.6),
+    "Atmospheric Radar S": otherElement("Atmospheric Radar S", Category.SENSOR, 96.56),
+    "Atmospheric Radar M": otherElement("Atmospheric Radar M", Category.SENSOR, 486.36),
+    "Atmospheric Radar L": otherElement("Atmospheric Radar L", Category.SENSOR, 2658.56),
+    "Space Radar S": otherElement("Space Radar S", Category.SENSOR, 96.56),
+    "Space Radar M": otherElement("Space Radar M", Category.SENSOR, 486.36),
+    "Space Radar L": otherElement("Space Radar L", Category.SENSOR, 2658.56),
+    "Static Core XS": otherElement("Static Core XS", Category.CORE_UNIT, 16.1),
+    "Static Core S": otherElement("Static Core S", Category.CORE_UNIT, 83.6),
+    "Static Core M": otherElement("Static Core M", Category.CORE_UNIT, 454.8),
+    "Static Core L": otherElement("Static Core L", Category.CORE_UNIT, 2501),
+    "Space Core XS": otherElement("Space Core XS", Category.CORE_UNIT, 14),
+    "Space Core S": otherElement("Space Core S", Category.CORE_UNIT, 120),
+    "Space Core M": otherElement("Space Core M", Category.CORE_UNIT, 420),
+    "Space Core L": otherElement("Space Core L", Category.CORE_UNIT, 1383),
+    "Dynamic Core XS": otherElement("Dynamic Core XS", Category.CORE_UNIT, 16.1),
+    "Dynamic Core S": otherElement("Dynamic Core S", Category.CORE_UNIT, 87.2),
+    "Dynamic Core M": otherElement("Dynamic Core M", Category.CORE_UNIT, 454.8),
+    "Dynamic Core L": otherElement("Dynamic Core L", Category.CORE_UNIT, 2501),
+    "Territory Unit": otherElement("Territory Unit", Category.CORE_UNIT, 4118.29),
+}
+
+/**
+ * Get the ContainerElements sorted by capacity from largest to smallest
+ */
+export const SORTED_CONTAINERS: ContainerElement[] = []
+for (const [key, val] of Object.entries(ITEMS)) {
+    if (isContainerElement(val)) {
+        SORTED_CONTAINERS.push(val)
+    }
 }
+SORTED_CONTAINERS.sort((a, b) => b.capacity - a.capacity)
```

Comment:
You can use the [`values`](https://ramdajs.com/docs/#values) function from `ramda` (or `Object.values` alternatively) to get the values of an object. Then you can filter that list using `isContainerElement` to only contain containers.

To sort a list of objects ascending by some property, you can use [`sortBy`](https://ramdajs.com/docs/#sortBy) and [`prop`](https://ramdajs.com/docs/#prop) from `ramda`.

```suggestion
export const CONTAINERS = values(ITEMS).filter(isContainerElement)
export const CONTAINERS_ASCENDING_BY_CAPACITY = sortBy(prop("capacity"), CONTAINERS)
```

Reply:
Done in https://github.com/lgfrbcsgo/du-factory-generator/commit/8f5a7f3b3faffc266ad88f3a885ec1c7a459fdca